examples/with-segment-analytics/pages/_app.js

```diff
@@ -1,8 +1,29 @@
 import Page from '../components/Page'
+import Script from 'next/script'
+import * as snippet from '@segment/snippet'
+
+function renderSnippet() {
+  const opts = {
+    apiKey:
+      process.env.NEXT_PUBLIC_ANALYTICS_WRITE_KEY ||
+      'NPsk1GimHq09s7egCUlv7D0tqtUAU5wa',
```

Comment:
Could we pull this into a variable to define what it is? Assuming it's a read-only token that's safe for demo usage or something?

Reply:
Ahh, I see it in the old diff

```
// This write key is associated with https://segment.com/nextjs-example/sources/nextjs.
```